erp/import_datasets/fetcher_strategy.py

```diff
@@ -0,0 +1,64 @@
+import csv
+import io
+from abc import ABC, abstractmethod
+from typing import Any, List, Iterable
+
+import requests
+
+
+class Fetcher(ABC):
+    fieldnames: List
+
+    @abstractmethod
+    def fetch(self, uri) -> Any:
+        pass
+
+
+class JsonFetcher(Fetcher):
+    def fetch(self, url):
+        try:
+            return requests.get(url).json()
+        except requests.exceptions.RequestException as err:
```

Comment:
We need to catch [json.JSONDecodeError](https://docs.python.org/3/library/json.html#json.JSONDecodeError) exceptions as well.